### PR TITLE
Update dependencies. Mainly, JRuby 9.2.7.0

### PR DESCRIPTION
**Pull request by justinstoller**

### CODE REVIEW

File: test/unit/puppetlabs/jruby_utils/slj4j_logger_test.clj [hunk 1]
```diff
@@ -23,7 +23,7 @@
     (testing "warn with a string and objects"
       (logutils/with-test-logging
        (.warn logger "a {} {} warning" (into-array Object ["strongly" "worded"]))
-       (is (logged? "a {} {} warning" :warn))))
+       (is (logged? "a strongly worded warning" :warn))))
```
Comment (Magisus) [inline test/unit/puppetlabs/jruby_utils/slj4j_logger_test.clj]: Did all these tests just fail before?

Comment (Magisus) [reply test/unit/puppetlabs/jruby_utils/slj4j_logger_test.clj]: Jk reading commit messages helps.

**Comment by justinstoller** [comment]
looks like I should update the travis.yml as well

**Comment by justinstoller** [comment]
This PR at b35a3b37d2e1ca062cb63ae1f2bb6b5f3e910bb3 passed the Puppet Server test suite locally for me (with the addition of having to specify snakeyaml explicitly).
```
Ran 186 tests containing 2598 assertions.
0 failures, 0 errors.
```

**Pull request merged**